escrow/payment_handler.go

```diff
@@ -60,14 +60,14 @@ func (h *paymentChannelPaymentHandler) Payment(context *handler.GrpcStreamContex
 
 	transaction, e := h.service.StartPaymentTransaction(internalPayment)
 	if e != nil {
-		return nil, paymentErrorToGrpcError(e)
+		return transaction, paymentErrorToGrpcError(e)
```

Comment:
Can transaction be a non-null object in this case? Since the error is from the StartPaymentTransaction method I assume that transaction would be nil if an error has occurred